packages/core/src/tools/create-logger.js

```diff
@@ -179,15 +186,20 @@ class LogStream extends Transform {
         'X-Token': token,
       },
       body: this,
+      signal: this.controller.signal,
     };
     return request(httpOptions).catch((err) => {
+      if (err.name === 'AbortError') {
+        return {
+          status: 200,
+          content: 'aborted',
+        };
+      }
+
       // Swallow logging errors. This will show up in AWS logs at least.
-      console.error(
-        'Error making log request:',
-        err,
-        'http options:',
-        httpOptions
-      );
```

Comment:
I'm removing `httpOptions` from the log because there were credentials in the AWS logs.